ruby_event_store/lib/ruby_event_store/serialized_record.rb

```diff
@@ -0,0 +1,13 @@
+module RubyEventStore
+  class SerializedRecord
+    def initialize(id:, data:, metadata:, event_type:)
+      @id         = id
```

Comment:
Do you think it would be better to rename it to `event_id` as higher layer use (or stay with `id` as lower layer uses?) I think `event_id` could be better.

Reply:
Do you think we should check that `id` is a string?

Reply:
event_id makes sense